extensions/amp-story/1.0/amp-story-quiz.js

```diff
@@ -0,0 +1,182 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {CSS} from '../../../build/amp-story-quiz-1.0.css';
+import {createShadowRootWithStyle} from '../0.1/utils';
+import {htmlFor} from '../../../src/static-template';
+import {isLayoutSizeDefined} from '../../../src/layout';
+
+/** @private {Array<string>} */
+const answerChoiceOptions = ['A', 'B', 'C', 'D'];
+
+export class AmpStoryQuiz extends AMP.BaseElement {
+  /**
+   * @param {!AmpElement} element
+   */
+  constructor(element) {
+    super(element);
+
+    /** @private {?ShadowRoot} */
+    this.shadowRoot_ = null;
+
+    /** @private {boolean} */
+    this.hasReceivedResponse_ = false;
```

Comment:
Nit: I know this rule is really not respected across our files as they grow, but on small files it'd be cool to A-Z these properties.

Reply:
+1 to this after the reviews :))